app/controllers/msa_components_controller.rb

```diff
@@ -7,13 +7,13 @@ def index
 
   def new
     @component = NewMsaComponentEvent.new
-    @hub_environments = Rails.configuration.hub_environments.keys
+    @hub_environments_legacy = Rails.configuration.hub_environments_legacy.keys
```

Comment:
you didn't have to change the variable name? you'll have to change it back later...

Reply:
we will!